### PR TITLE
editoast: add utoipa annotation for `infra/{id}/lock`

### PR DESCRIPTION
**Pull request by hamz2a**
Part of #4580

### CODE REVIEW

File: editoast/openapi.yaml [hunk 1]
```diff
@@ -8686,6 +8671,25 @@ paths:
       summary: Returns the BBoxes (geo and schematic) of a line
       tags:
       - infra
+  /infra/{infra_id}/lock/:
+    post:
+      description: Lock an infra
+      parameters:
+      - description: The ID of the infra to fix
```
Comment (leovalais) [inline editoast/openapi.yaml]: Better change the comment in InfraIdParam to `An existing infra ID`

**Review by leovalais — APPROVED** [review]
Thanks!

File: editoast/openapi.yaml (hunk 1 above)
Comment (hamz2a) [reply editoast/openapi.yaml]: Done.

**Review by SharglutDev — APPROVED** [review]
Lgtm

**Pull request merged**